lib/views/file-patch-view.js

```diff
@@ -228,7 +228,8 @@ export default class FilePatchView extends React.Component {
         buffer={this.props.filePatch.getBuffer()}
         lineNumberGutterVisible={false}
         autoWidth={false}
-        autoHeight={false}
+        // TODO only set to true for commit previews, but not for single FilePatchViews
+        autoHeight={true}
```

Comment:
Would it be possible to disable `autoHeight` again for "single" `FilePatchView`s? Currently they don't scroll anymore.

Reply:
Fixed ✅  I've added a prop to `FilePatchView` that controls whether or not `autoHeight` is set and set it appropriately in the parent components.